pkg/test-infra/tidb/recommend.go

```diff
@@ -110,10 +107,10 @@ func RecommendedTiDBCluster(ns, name string) *Recommendation {
 					},
 				},
 				TiKV: v1alpha1.TiKVSpec{
-					Replicas:             *TiKVReplicas,
-					ResourceRequirements: fixture.WithStorage(fixture.Medium, "10Gi"),
+					Replicas:             int32(fixture.Context.TiKVReplicas),
+					ResourceRequirements: fixture.WithStorage(fixture.Medium, "100Gi"),
```

Comment:
Should this be `100Gi`? Or do we want this to be configurable?

Reply:
Yes, we can make it configurable on another PR.